src/App.js

```diff
@@ -1,5 +1,105 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+  constructor() {
+    this.answerNumber = '';
+    this.playerInput = '';
+  }
+  // error 핸들링 메서드
+  handleError(message) {
+    let error = new Error();
+    error.message = message;
+    Console.print(error.message);
+    throw error;
+  }
+  // 랜덤 숫자 생성 메서드
+  setAnswerNumber() {
+    const numbers = [];
+    while (numbers.length < 3) {
+      const number = Random.pickNumberInRange(1, 9);
+      if (!numbers.includes(number)) {
+        numbers.push(number);
+      }
+    }
+    this.answerNumber = numbers.join('');
+  }
+  // 입력값이 검증 메서드
+  validateInput(input) {
+    const isDuplicate = input.split('').some((number) => input.indexOf(number) !== input.lastIndexOf(number))
+
+    if(isNaN(Number(input))) {
+      this.handleError('[ERROR] 숫자를 입력해주세요');
+    } else if(input.length !== 3) {
+      this.handleError('[ERROR] 3자리 숫자를 입력해주세요');
+    } else if(input.includes('0')) {
+      this.handleError('[ERROR] 1~9 사이의 숫자를 입력해주세요');
+    }  else if(isDuplicate) {
+      this.handleError('[ERROR] 중복되지 않는 숫자를 입력해주세요');
+    } else {
+      this.playerInput = input;
+    }
+  } 
+  // 볼, 스트라이크 판단 메서드
+  calculateBallsAndStrikes() {
+    const player = this.playerInput.split('');
```

Comment:
여기도 spread syntax를 사용해볼 수 있을 것 같네요!
```suggestion
    const player = [...this.playerInput];
```